rust/variable/variable.rs

```diff
@@ -41,10 +41,10 @@ impl Variable {
         }
     }
 
-    pub fn is_name(&self) -> bool {
+    pub fn is_named(&self) -> bool {
```

Comment:
`is_name()` is from the now removed `Reference`. `is_named()` makes more sense on a `Variable`.

Reply:
Ok, this is fine